components/micro_wake_word.rst

```diff
@@ -26,8 +26,9 @@ The training process is described on the `microWakeWord GitHub repository <https
 Configuration variables:
 ------------------------
 
-- **microphone** (**Required**, :ref:`config-microphone-source`): The :doc:`microphone </components/microphone/index>` settings to use for input.
-- **models** (**Required**, list): The models to use.
+- **microphone** (**Required**, :ref:`config-microphone-source`): The :doc:`microphone </components/microphone/index>` settings to use for audio input.
+- **stop_after_detection** (*Optional*, boolean): Whether to stop the component after detecting a wake word. Defaults to ``true``.
+- **models** (**Required**, list): The models to use. Only the first model is enabled by default on the first boot. Each model's enabled state is then saved/restored to/from the flash.
 
   - **model** (**Required**, string): This can be one of:
```

Comment:
```suggestion
  - **id** (*Optional*, :ref:`config-id`): The optional ID used for the model actions below.
  - **model** (**Required**, string): This can be one of:
```